localisation/000_wc_traits.csv

```diff
@@ -20,7 +20,7 @@ explorer_desc;Explorers are those brave souls who risk their lives for the sake
 creature_naga;Naga;Naga;Naga;;;;;;;;;;;x
 creature_naga_desc;The naga little resemble the nobility they once were. Their proud appearance has been warped and distorted, many of them received a number of special changes characteristical of the sea depths, such as an ability to release ink, tentacles, sharp scales, poisonous fangs, strong shells of mollusks, serpents instead of hair, large and sharp dorsal fins, and strong tails. In ancient times a matriarchy was established, when the witches and sirens developed stronger intellect and magical capabilities to control the entire community of naga. Males, who lost most of their magical powers during their transformation, acquired inhuman physical strength, and now constitute the main strike force of naga armies. In general, their exile has only strengthened the evil, vicious nature of these naked and sadistic creatures.;;Die Naga �hneln wenig dem Adel, der sie einst waren. Ihr stolzes Aussehen wurde verzerrt und entstellt, viele von ihnen erhielten eine Reihe von besonderen Ver�nderungen, die f�r die Tiefen des Meeres charakteristisch sind, wie z.B. die F�higkeit, Tinte, Tentakel, scharfe Schuppen, giftige Rei�z�hne, starke Muschelschalen, Schlangen anstelle von Haaren, gro�e und scharfe R�ckenflossen und starke Schw�nze freizusetzen. In der Antike wurde eine matriarchale Gesellschaft gegr�ndet, als die Hexen und Sirenen einen st�rkeren Intellekt und magische F�higkeiten entwickelten, um die gesamte Gemeinschaft der Naga zu kontrollieren. M�nner, die w�hrend ihrer Transformation die meisten ihrer magischen Kr�fte verloren haben, erwarben unmenschliche k�rperliche Kr�fte und bilden heute die Hauptstreitkraft der Naga-Armeen. Im Allgemeinen hat ihr Exil die b�se, hinterh�ltige Natur dieser nackten und sadistischen Kreaturen nur noch verst�rkt.;;;;;;;;;;;x
```

Comment:
```suggestion
creature_naga_desc;The naga little resemble the nobility they once were. Their proud appearance has been warped and distorted, many of them received a number of special changes characteristic of the sea depths they now call their home, such as an ability to release ink, tentacles, sharp scales, poisonous fangs, strong shells of mollusks, serpents instead of hair, large and sharp dorsal fins, and strong tails. The naga adopted a matriarchal system in ancient times, as the witches and sirens developed stronger intellect and magical capabilities to control the entire community of naga. Males, who lost most of their magical powers during their transformation, acquired inhuman physical strength, and now constitute the main strike force of naga armies. In general, their exile has only strengthened the cruel nature of these sadistic creatures.;;Die Naga �hneln wenig dem Adel, der sie einst waren. Ihr stolzes Aussehen wurde verzerrt und entstellt, viele von ihnen erhielten eine Reihe von besonderen Ver�nderungen, die f�r die Tiefen des Meeres charakteristisch sind, wie z.B. die F�higkeit, Tinte, Tentakel, scharfe Schuppen, giftige Rei�z�hne, starke Muschelschalen, Schlangen anstelle von Haaren, gro�e und scharfe R�ckenflossen und starke Schw�nze freizusetzen. In der Antike wurde eine matriarchale Gesellschaft gegr�ndet, als die Hexen und Sirenen einen st�rkeren Intellekt und magische F�higkeiten entwickelten, um die gesamte Gemeinschaft der Naga zu kontrollieren. M�nner, die w�hrend ihrer Transformation die meisten ihrer magischen Kr�fte verloren haben, erwarben unmenschliche k�rperliche Kr�fte und bilden heute die Hauptstreitkraft der Naga-Armeen. Im Allgemeinen hat ihr Exil die b�se, hinterh�ltige Natur dieser nackten und sadistischen Kreaturen nur noch verst�rkt.;;;;;;;;;;;x
```

Reply:
done